src/ck-core/CMakeLists.txt

```diff
@@ -103,6 +105,8 @@ add_library(moduleDistributedLB ../ck-ldb/DistributedLB.C)
 add_dependencies(moduleDistributedLB ck)
 add_library(moduleTreeLB ../ck-ldb/TreeLB.C)
 add_dependencies(moduleTreeLB ck)
+add_library(moduleRecBipartLB ../ck-ldb/RecBipartLB.C)
```

Comment:
```suggestion
add_library(moduleRecBipartLB ../ck-ldb/RecBipartLB.C ../ck-ldb/RecBipartLB.h)
```

Reply:
Aside from this change, has my +1 for the CMake part.

Reply:
Hmm, should the other LBs listed above then also have their headers listed as sources?

Reply:
In principle yes, but it's not a big deal. cmake determines what to do with a file based on its extension. For .h files, it just creates a dependency between the file and the target (in this case, if  ../ck-ldb/RecBipartLB.h changes, recompile moduleRecBipartLB)